frontend/components/Players/PlayerManager.vue

```diff
@@ -378,20 +369,67 @@ export default Vue.extend({
     },
     handleKeyPress(e: KeyboardEvent): void {
       if (!this.isMinimized && this.isPlaying) {
+        const focusEl = document.activeElement;
+
+        let spaceEnabled = false;
+
+        if (e.key === 'Spacebar' || e.key === ' ') {
+          spaceEnabled =
+            focusEl?.classList.contains('v-dialog__content') ||
+            focusEl?.classList.contains('hide-pointer') ||
+            focusEl?.className === '';
+        }
+
         switch (e.key) {
           case 'Spacebar':
           case ' ':
+            if (spaceEnabled) {
+              this.playPause();
+            }
+
+            break;
+          case 'k':
             this.playPause();
             break;
           case 'ArrowRight':
+          case 'l':
             this.skipForward();
             break;
           case 'ArrowLeft':
+          case 'j':
             this.skipBackward();
+            break;
+          case 'f':
+            if (this.getCurrentlyPlayingMediaType === 'Video') {
+              this.toggleFullScreen();
+            }
+
+            break;
+          case 'm':
+            this.toggleMute();
+            break;
+        }
+      } else if (this.isMinimized && this.isPlaying) {
+        switch (e.key) {
+          case 'f':
+            if (this.getCurrentlyPlayingMediaType === 'Video') {
+              this.toggleMinimized();
+            }
+
             break;
         }
       }
     },
+    handleVideoClick(e: MouseEvent) {
+      const target = e.target as HTMLElement;
+
+      if (
+        target?.classList?.contains('player-overlay') &&
```

Comment:
```suggestion
      const target = e.target;

      if (
        target &&
        target.classList.contains('player-overlay') &&
```

This way you have proper checks at runtime

Reply:
Btw, no luck at binding this to the ``playPause`` method [here](https://github.com/jellyfin/jellyfin-vue/blob/824c7008071e2e0335569883ae57cc9775ffb730/frontend/components/Players/PlayerManager.vue#L31)?

Reply:
Hi @ferferga, sorry if I answer you late but I was having trouble installing dependencies from the structure change ... (a TLS problem with npm, which I solved).

Ah anyway, I tried (and had tried at the time) to directly insert the playPause on the click of the 'v-overlay' as you suggest.
The problem is that it is fired all the time (even when you press anything else on the screen, with nasty side effects).

Using the click handler and checking the 'player-overlay' class, I am able to obtain only the significant clicks (i.e. those in the purple area of the photo, avoiding any possible side effect (like double action @ once)), 
_which in my opinion results in the best experience._

![image](https://user-images.githubusercontent.com/61033694/155706870-95ab0bf0-0a29-4b95-8b2e-313b82f81ec5.png)

Reply:
Okay, add the type checks to that section and all fine to me.

Reply:
Oh sure, forgotten.

Reply:
@ferferga, 
I made the change, however I have to keep "as HTMLEvent" because MouseEvent declares target as an EventTarget which does not hold the properties of an HTMLElement, however it is an HTMLElement that I receive (and that i need).

The alternative, but less elegant, would be to type a: any target, but it didn't seem the best...

Look good to you?